packages/zowe-explorer-ftp-extension/CHANGELOG.md

```diff
@@ -8,6 +8,12 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 
 ### Bug fixes
 
+## `2.6.2`
```

Comment:
Have we released a 2.6.2 patch? I don't recall us doing so - should this say `TBD Release`?

Reply:
we discussed the 2.6.2 release for security updates with [PR](https://github.com/zowe/vscode-extension-for-zowe/pull/2156) that got merged to maintenance this week.  I also left a message in our slack about preparing that release in the week to come.  We don't have to merge this into main until the 2.6.2 goes out if preferred.